src/components/Header/index.js

```diff
@@ -1,18 +1,35 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 
+import { Nav, Link } from './styled'
 import Urls from '../../constants/urls'
 
+const Links = [
```

Comment:
We use uppercase first letter only for functional component constants and for stateful component classes.
It's kind of convention to use constants like this with lowercase first letter (same as in case of `src/constants/urls`). So when you see upper cased first letter, you know that it is component.

Reply:
Yeah, you are right. Thanks for notice. I must write this in to my head.